tests-beta/android/MASVS-STORAGE/MASTG-TEST-0216.md

```diff
@@ -9,29 +9,18 @@ best-practices: [MASTG-BEST-0004]
 
```

Comment:
Since it's solely a dynamic test now, shouldn't we start this title with "Runtime use ..."? Not sure if we can find a good title with this prefix but how about "Runtime Testing for Presence of Sensitive Files in Backups"

Reply:
Good point, and I thought about it. However, I began to see a pattern of tests that did not fit our very well-intentioned naming rule.

See here: https://github.com/OWASP/owasp-mastg/pull/3242/files#diff-4dcf4f4e02c7e27d435bcc8d6c096a0c351a07fb9caf7f8ac301d3dd59c91c40

This is another _dynamic_ test where it would be weird to start it with "Runtime ...". I propose we keep it consistent for the typical static (semgrep/r2) and dynamic (frida) and let's see how this other kind evolves and if we come up with a good convention.